process/project-lifecycle-documents/sci_sandbox_stage.md

```diff
@@ -0,0 +1,40 @@
+## Application for creating a new project at Sandbox stage
+
+### List of project maintainers
+
+The project has [4 maintainers](https://github.com/revanite-io/sci/graphs/contributors) from 4 different companies:
+
+* Eddie Knight, Sonatype, @eddie-knight
+* Travis Truman, Independent, @trumant
+* Jason Meridth, GitHub, @jmeridth
+* Alex Speasmaker, USAA, @speas038
```

Comment:
```suggestion
The project has [4 maintainers](https://github.com/revanite-io/sci/graphs/contributors) from 4 different companies:

* Eddie Knight, Sonatype, @eddie-knight
* Travis Truman, Independent, @trumant
* Jason Meridth, GitHub, @jmeridth
* Alex Speasmaker, USAA, @speas038
* Damien Burks, Citi, @damienjburks
```

Reply:
The more maintainers the better, but I didn't see Damien listed in https://github.com/revanite-io/sci/graphs/contributors

Reply:
Hah, yeah his GH invite is pending but he did confirm verbally - I didn't click the button on it until he was logged out for the weekend. We can wait to apply this after the GH group is updated.